src/draw/vg_lite/lv_draw_vg_lite_label.c

```diff
@@ -239,14 +254,26 @@ static void draw_letter_bitmap(lv_draw_vg_lite_unit_t * u, const lv_draw_glyph_d
         lv_vg_lite_path_drop(u, path);
     }
 
-    /* TODO: The temporary buffer of the built-in font is reused.
-     * You need to wait for the GPU to finish using the buffer before releasing it.
-     * Later, use the font cache for management to improve efficiency.
-     */
+#if LV_FONT_FMT_TXT_CACHE_GLYPH_CNT > 0
```

Comment:
If we are rendering a FreeType or TinyTTF font it will be cached because of `LV_FONT_FMT_TXT_CACHE_GLYPH_CNT`. So it seems seems more like a generic bitmap font cache, not related only to the built-in format.

Reply:
At present, both freetype and tinyttf support cache reference counting. Later, we can consider centralized management of font cache, and the font back-end only needs to focus on adapting the creation method of font bitmap.
At present, we can first step out of the built-in font independent cache, and gradually rebuild and modify it later :)

Reply:
I see, but it feels a little messy to me. What about this?
- Keep everything as it is in this PR, just change the name from `LV_FONT_FMT_TXT_CACHE_GLYPH_...` to `LV_FONT_CACHE_GLYPH_...` to make it sound generic. 

In a follow up PR we can remove the TinyTTF and FreeType bitmap caches.

Reply:
Done.

Reply:
Thanks!